service/gcsutils/libtar2vhd/tar2vhd.go

```diff
@@ -80,7 +80,7 @@ func VHD2Tar(in io.Reader, out io.Writer, options *Options) (int64, error) {
 		return 0, err
 	}
 
-	mntFolder, err := ioutil.TempDir("", "mnt")
```

Comment:
the first argument of `ioutil.TempDir` is the directory location, so it should be `ioutil.TempDir("tmp", <whatever>)`